microsoft/botframework-solutions
# [4.4] Added va_add_remote_skill.ps1

## Description
- Added the add_remote_skill.ps1 script

## Testing Steps
1. Change to the project directory
2. Run `..\Deployment\Scripts\va_add_remote_skill.ps1 -configFile skills.json -manifestUrl https://bfcalendarskill.azurewebsites.net/api/skill/manifest -dispatchName DISPATCH_NAME`

By default, the dispatch files should be in the Deployment\Resources\Dispatch\{LanguageCode} folder, but you can provide the -dispatchFolder parameter if it's somewhere else
